### DISCUSSION TITLE
Meeting Notes Process Discussion

### DISCUSSION THREAD

**Discussion by santanainniss** [WordPress/Marketing-Team#148] [General]
There have been many conversations and GitHub issues opened in 2022 related to note taking, pain points for contributors, and how to make the notes easier for contributors. Some examples of this include [GitHub Issue 68](https://github.com/WordPress/Marketing-Team/issues/68), [GitHub Issue 103,](https://github.com/WordPress/Marketing-Team/issues/103), and these Slack threads: [Thread 1](https://wordpress.slack.com/archives/C0GKJ7TFA/p1669822936218659), [Thread 2](https://wordpress.slack.com/archives/C0GKJ7TFA/p1662565856018919), [Thread 3](https://wordpress.slack.com/archives/C0GKJ7TFA/p1664956621412719). 

Some common pain points include:

-Granting new contributors blog access/new contributors unsure how to navigate the blog for note taking, following best practices in the blog e.g. tagging the notes, etc 
-Using Slack, copy/pasting Slack links into the notes 
-Contents of notes: linking contributor ideas, linking Slack threads, how much of the meeting include in the notes etc, structure of the notes 
-Consistency of notes vs the agenda 
-Time commitment: amount of time to create/draft notes, review and editing of notes, time between meeting and posting of the notes.

Make Marketing contributors have gone above and beyond to help onboard new contributors to the note taking process: hosting Zoom training calls, requesting screenshots and or videos be made help educate new contributors, and many edits to the handbook page. Despite this meaningful work, contributors new and seasoned continue to struggle with these pain points. 

This made me wonder: are we solving the right problem? I looked around at the notes of various Make teams, and found a few important differences between their process for note taking and our own. Here are some of those differences:

- Make Marketing notes include far more Slack links than many other make teams. Where Marketing tends to link each item from the agenda and much of the content of the meeting itself. Teams like [Openverse](https://make.wordpress.org/openverse/2022/12/02/community-meeting-recap-29-november-2022/) and the [Diversity Speaker Training (Community) Team](https://make.wordpress.org/community/2022/11/24/recap-of-the-diverse-speaker-training-group-wpdiversity-amer-emea-on-november-23-2022/) link only stat and end points. The [Docs ](https://make.wordpress.org/docs/2022/11/30/summary-for-docs-team-meeting-november-29-2022/) and [Photos](https://make.wordpress.org/photos/2022/06/17/recap-of-team-meeting-june-2/) teams list only starting point Slack link. 
-Make Marketing notes tend to contain a lot more detail and complexity, whereas the linked examples above include only brief summaries. 

I'd like to open a conversation with all members of the Community to discuss how we can make the note taking process simpler and less complex. Restructuring our notes and processes to be closer to those of other Make teams can help us reduce the need for continual trainings and complex documentation.

**Comment by DanSoschin** [comment]
I agree! Note-taking should be quick and easy. I imagine that an attendee of one of the weekly meetings should be able to compile and publish notes in about 10 minutes. Providing 5-10 bullet points as a template seems sufficient as well. There shouldn’t need to be any training or videos for note-taking, it’s something that we’ve all done at some point in our lives, and the prior published meeting notes should be sufficient for serving as a guide/template. I’d prefer someone spend their time not on lengthy or complicated meeting notes, but rather helping amplify the WP mission on social media, for example. That activity would produce more ROI than a complicated note process, in my opinion... The more we can do to make things simple, the better.

**Comment by abhansnuk** [comment]
[Edited comment due to broadband going down inbetween]

Continuing to make things easier is a shared aim, and good that we are reviewing this on a regular basis.

Some contributors have found adding notes easy and take 15 minutes to write, including some of the newer contributors. I agree that we do not need a Slack link for every agenda item, but they might be more useful for those where there is a summary of a long discussion and link back to it for the full details. The beginning of the Slack meeting is the most useful link for notes. 

I would suggest two things:
- that if there is a significant item we want a wider audience to be aware of and potentially get involved with too, it is useful to have a summary in the P2 without just the top level of agenda item and then requiring access to Slack for the detail. What we have been advised previously is to avoid people already being on Slack to get key information. The GitHub cards address this partly as they are in the open and they are linked from the P2.
- be aware that for many contributors, English may not be their first language, and any help with doing notes needs to provide support for this in the way that supports them most. If generic information is not enough for the particular contributor, then help can be provided in alternative formats that they request, if it is possible. 

For clarity, the training has been about using the P2 blog for those unfamiliar with it, where to find categories and tags, and adding IDs in the way the P2 blog uses it, how to add the beginning of the meeting slack link. These have been suggested to be added to the notes in the handbook as screenshots as these were people regularly have difficulties. The video screenshare as an interim has been offered if people had said they were not familiar with using WordPress and have felt they would prefer that. Screenshots of categories and tags would be helpful.

**Reply by santanainniss** [reply]
>  if people had said they were not familiar with using WordPress

I hear that, and value all the openness you've shown to help folks with this. I am wondering about the **should** of this. Should the note taking responsibility be a first issue if a contributor does not have familiarity with WordPress? 

I ask this because I have observed that the notes process is not considered a "good first issue" with the other Make teams I've surveyed. Are we trying to fit square peg into a round hole? I would rather invest is creating good first issues that help new contributors start off feeling empowered. It is a BIG ask for a first issue-- Volunteer, have some one add you to a blog you may or may not know how to use, get training on managing the blog-- all before you can actually create notes. It's a big lift and it may deter contribution.

**Reply by abhansnuk** [reply]
I agree, I think notes can be a difficult first good issue in most teams. I think having more good first time issues would be a lot more useful. From experience in the team, the P2 itself tends to be the biggest challenge for some volunteers who have not used the CMS in the way it displays in the P2.

More information was added to notes in direct response to contributors saying they could not follow minimal notes had before or used in examples, especially if they were new to the team. We have a lot of what feels like duplication at the moment, so cutting some of that and getting the balance as the team continues to grow. Some of the people who tested notes in their organizations also commented that just giving a slack link to discover the summary was unhelpful and another hoop to jump through. So even if it is a very brief summary, and then linking to something like a P2 post or a GitHub card, which does not require logging on, would help address this.

**Comment by thetinyl** [comment]
Thanks for opening this discussion @santanainniss. 

Something I noticed about the examples from other teams that were shared above, is how quick and easy their notes are to scan through. 

Ours do spend a lot of time contextualizing and educating on the meeting process, which can sometimes get in the way of the main point of the information being shared. I wonder if paring down a little on that (or providing a short version of that information upfront or linking to it elsewhere) might also help simplify the process. 

To @abhansnuk's point: 

> if there is a significant item we want a wider audience to be aware of and potentially get involved with too, it is useful to have a summary in the P2 without just the top level of agenda item 

I'm curious if condensing some of the explanations within the notes might help make these instances stand out. 

Overall, without being able to clearly see what information is different at a glance, it's easy to think the notes are a lot of the same thing each meeting. I've found the extent of the repetition makes copy editing a bit of a challenge as well when taking notes.

**Reply by jpantani** [reply]
Yes, less repetition of cut, copy, and paste and more interaction and engagement with the substance of the meetings!

**Reply by santanainniss** [reply]
I'm in favor of condensed notes. We can certainly shave of complexity by removing repetition and the links to every Slack thread,

**Reply by abhansnuk** [reply]
Yes I think condensing some of the explanations would help them stand out. Definitely favor less copy and paste from the meeting, and more highlighting of the substance. 

Summaries from feedback I have read or heard from others is not something that all contributors may find easy to do. When there have been summaries in the past, including when I started out, it can be easy to get the wrong end of the stick in the summary if you are new to the project or unfamiliar with what is being discussed. Yvette and Maedah previously supported new contributors with explaining some of the WP terms and an intro to WP. More recently, Learn WP has a lot of material that can be helpful too.

**Comment by jpantani** [comment]
Great discussion folks! Thanks for bringing it up @santanainniss 

For me, I think it would be wonderful if the notes were synthesized going forward. I am completely overwhelmed when I see the notes with information overload. I think it would be great if they were in fact short and sweet and served as a way to send people to conversations, topics, and DRIs for the various initiatives. 

I don’t think that would require native English speakers because notes don’t need to be in perfect English, grammar, or punctuation. Plus, they're just notes. We should be able to scan them in a simple way. So I support a change in how we do them. It's a great way to get started with the Make Team so doing them shouldn't feel overly burdensome.

Here’s a snippet of what I propose. 

1) The team welcomed A,B,C, and D. [New entries should be @ mentioned to reinforce a sense of being welcomed and belonging.]
2) The social question was “Do you like ’thing?’” Add your comment here [slack link]
3) Topics/Agenda items discussed — please join the conversation in slack or Github! 

- Topic 1 - brief description in 10 words or less of topic (Slack; GitHub)
- Topic 2 (Slack; GitHub)
-           -Topic 2 Follow up action (this could be a sub-bullet) 
- Topic 3 (Slack; GitHub)

4) Open floor
           (no topics were raised this week or we discussed Notes here [link])
5) Thank you for joining

I also observed in the comments above the issue of not having enough context in the marketing P2 where the notes currently resolve. I think that invites more contributions to talk about initiatives in the Make Marketing P2. As an example, I posted about Grow Your Story there. If we have important items they should at some point be P2'ed :) In my humble opinion that seems an ideal place to add what we feel is most important and plan on devoting significant energy, time, and resources.

**Reply by santanainniss** [reply]
So much good stuff here, Jonathan.

> I don’t think that would require native English speakers because notes don’t need to be in perfect English, grammar, or punctuation.

I agree with this 100%. These are notes, not marketing material. We don't need to be perfect with notes. I'm always in favor of inclusion-- including inclusion for non-native speakers. If there's ever a time when something is really unclear in notes, I'm sure a team rep could do a light edit for clarity. 

> Here’s a snippet of what I propose.

I'm thinking we could simplify this even further. For example, our social questions are a fun way to bond. But I'm not sure how helpful they are in notes. 

For me, the contribution and participation we really want is in Slack where the meeting takes place. I view the notes as a way for folks to get a super quick overview of the meeting that drives them to open up Slack if there's an important item.

> In my humble opinion that seems an ideal place to add what we feel is most important and plan on devoting significant energy, time, and resources.

I think our P2 is our most underutilized resource. I love to see us start using it more in the New Year!

**Comment by murillol** [comment]
> Ours do spend a lot of time contextualizing and educating on the meeting process, which can sometimes get in the way of the main point of the information being shared. I wonder if paring down a little on that (or providing a short version of that information upfront or linking to it elsewhere) might also help simplify the process.

I like the idea of putting all that info in a single post which we then link out to @thetinyl 

> Notes don’t need to be in perfect English, grammar, or punctuation.

This is true, although I'd like to note that non-native speakers have asked for more guidance and attention and it's not always about English or grammar, just requesting someone to talk them through it, look at it before they submit, etc.

>  If we have important items they should at some point be P2'ed

Agree!

> Here’s a snippet of what I propose.

I like this outline @jpantani! I'd rethink item # 1 about welcoming individually - people leave and re-join the channel _a lot_, and it's not always easy to find people's WP IDs which can make it cumbersome for the note-taker.

**Reply by jpantani** [reply]
Good point about the comings and goings of folks @murillol  - it might be good just to write a warm, heartfelt welcome note (that doesn't sound scripted if possible!) 

> This is true, although I'd like to note that non-native speakers have asked for more guidance and attention and it's not always about English or grammar, just requesting someone to talk them through it, look at it before they submit, etc.

If people are nervous about this it might make more sense to have last week's notetaker take a quick look at this week's notes for the current notetaker. Sort of like a guide who helps give confidence in the notes/notetaking process. This way we have the documentation but also add a human element that people might find lacking.  

> If we have important items they should at some point be P2'ed
> 
> Agree!

This is a muscle we need to train collectively. Would be wonderful to see that be a more intentional practice in 2023!

**Reply by abhansnuk** [reply]
More use of the P2 would be great, especially if there was a plan so people had a chance to contribute to a forthcoming post itself or suggest ideas. @sereedmedia and others have highlighted something similar before too. so including her in this comment. I would be good to have clarity on who can create a P2 post for a discussion as this has been limited previously, and there may be uncertainty on this and partly why it tends to have limited use for discussions/ initiatives from wider team members. There was a GitHub card set up to try and encourage discussion on topics for the P2. Perhaps doing something like this more actively would help get the P2 into greater use.

If we can avoid saying 'new' joiners/ contributors to marketing, this would be positive. As Larissa has said, some contributors only stay in the channel for specific items, and come and go. There have been instances where contributors have become upset at being welcomed as new when they have contributed to the channel over many years, or when a comment has been made about them leaving, when they simply do not keep all channels open on their slack. It is really hard for a notetaker or someone welcoming someone into the team to know if someone is new to marketing, to the project, or a returner. The approach we used before was to try and make sure that one of the team reps at least said hello or added an emoji to someone coming into the channel. We also had two people in the team who followed up with new people.

**Comment by santanainniss** [comment]
I'm noticing some other important topic surface here, which I just want to name. To recap the discussion so far, I'm hearing that using our P2 more often for our important items is a shared desire. I'm hearing a general agreement that the notes/note process can and should be simplified. And I'm also noticing challenges that new note takers, particularly non-native English speakers face-- for example, unfamiliarity with P2 or wanting someone to review notes before posting.

**Comment by santanainniss** [comment]
We could make some radical changes, as well. I really loved the simplicity of the Openverse team notes (linked above), so I reached out to ask how they handle their process. It doesn't seem like new contributors are offered note taking as a first issue at all. Instead, the responsibility rotates around reps and more seasoned contributors. We could do the same. 

At the same time, our team doesn't have a lot of meaning 'good first issue' opportunities to get new contributors onboarded quickly. My next questions _how can we make the note process an actual good first issue?_ One thing I am curious about is why we stopped allowing contributors to submit notes in a Google doc? Does anyone have some context or background they could share about this? If contributors can share simplified notes in another format-- we would not need to create new P2 accounts or train on the P2 for first time note takers. If a person continues with notes and want to take on the responsibility of direct posting in the future, we can cross that bridge then.

**Reply by abhansnuk** [reply]
It was @eidolonnight's idea to cut out the Google doc step to make it easier and one less step. This works for contributors who are happy to type in the P2 directly and are familiar with the CMS. It avoids some of the formatting issues when you bring google docs information into the P2. But on the rota there have been several weeks where contributors have not liked typing into the P2, and in those cases, when they have asked for help or someone to check through their notes, they have preferred for this to be in Google docs first. In some of these instances, the contributor has asked for help using the P2 too.

So perhaps continue with what works for the particular contributor. Some have moved onto using the P2 directly after receiving help with using it, and increased confidence about writing notes in English.

**Reply by eidolonnight** [reply]
> It was @eidolonnight's idea to cut out the Google doc step to make it easier and one less step.

Correct. We asked contributors to draft notes in WordPress to avoid the added work that comes from transferring Google docs into WordPress posts. Having everyone write directly in the Make blog reduced time spent on notes and gave contributors experience with the WordPress editor, experience that every WordPress marketer should have.

**Comment by santanainniss** [comment]
There has been a lot of great discussion here so far! Props to all our wonderful contributors. 

In our continued effort to conclude outstanding discussions and issues, Team Reps are issuing a final call for contribution on this discussion thread. Please be sure to get your final comments/thoughts/conversation in by **Sunday, 5 February**

After the call for contribution ends on 5 February, Team Reps will synthesize the discussion and identify a solution to pilot for a limited time period. After this test period, we'll analyze the impact of our test and open another discussion for feedback. We'll iterate on the pilot period as needed together in collaboration. 

Thank you again!

**Comment by jpantani** [comment]
Thanks for all the wonderful discussions on the topic of notes. 
With the 6.2 release underway, it's been harder for me to attend synchronously so I have been using the notes to help me catch up.
So I am using the notes like the ones from last week as a litmus test for my thoughts: https://wp.me/p72A3H-2LV

I've noticed there are a lot of long links. 
The shortlink feature might help alleviate some of this strain but this might also be a personal preference. In Core posts and /news announcements I've seen as a practice to hyperlink directly in context.

I've also noticed that adding events that have passed in the notes may no longer be timely.
> It is WordPress 6.2 Beta 1 day
Perhaps this can be omitted or stated "Feb 7 was Beta 1 was announced."

I love the new synthesis but I also see a slight opportunity for a bit more clarity; I've added an example below which I have found to be super helpful when I provide updates and summaries. 

> 
> **Team description —** The marketing description is being revisited with project leadership to ensure it aligned with the WordPress project goals, including readability: https://github.com/orgs/WordPress/projects/21
> **New Notes format —** We are testing a new notes format for the next two months based on the input received on #148: https://github.com/WordPress/Marketing-Team/discussions/148
> **Onboarding new contributors —** How can we help onboard our new contributors, especially in light of WordCamp Asia’s upcoming Contributor Day? https://github.com/WordPress/Marketing-Team/issues/169
> **Marketing team goals —** Ongoing discussion on the goals for the 2023 Marketing team. Join the discussion on GitHub card #165: https://github.com/WordPress/Marketing-Team/discussions/165
> 

Other than that the new notes are a breath of fresh air — thank you all involved in this shift!

**Comment by santanainniss** [comment]
Thank you to everyone that has joined into this discussion. After consolidating all feedback, the Team Reps are now piloting a more pared back version of the notes. (see example: https://make.wordpress.org/marketing/2023/02/21/notes-for-the-global-marketing-team-meeting-14-february-2023/). Please follow along with notes posted over the next two months to get a feel for the new appraoch, and comment you feedback directly on notes posts. 

Shipping this https://make.wordpress.org/marketing/2023/02/21/notes-for-the-global-marketing-team-meeting-14-february-2023/ <-- with an approach that consolidates the discussion above as much as possible. We will iterate again after a period of testing and a new call for feedback.
